docs/configuration/acme.md

```diff
@@ -112,21 +110,21 @@ entryPoint = "https"
   #
   entryPoint = "http"
 
-# Use a DNS-01/DNS-01 acme challenge rather than HTTP-01 challenge.
-# Note : Mandatory for wildcard certificates generation.
+# Use a DNS-01 ACME challenge rather than HTTP-01 challenge.
+# Note: mandatory for wildcard certificate generation.
 #
 # Optional
 #
 # [acme.dnsChallenge]
 
-  # Provider used.
+  # DNS management provider used.
   #
   # Required
   #
   # provider = "digitalocean"
 
   # By default, the provider will verify the TXT DNS challenge record before letting ACME verify.
-  # If delayBeforeCheck is greater than zero, avoid this & instead just wait so many seconds.
+  # If delayBeforeCheck is greater than zero, this is avoided and the given duration in seconds is waited.
```

Comment:
`If delayBeforeCheck is greater than zero, this check is delayed for the configured duration in seconds.`